docs/install.md

```diff
@@ -477,6 +479,32 @@ the `feature-flags` ConfigMap alongside your Tekton Pipelines deployment via
 
 For beta versions of Tekton CRDs, setting `enable-api-fields` to "beta" is the same as setting it to "stable".
 
+## Enabling larger results using sidecar logs
```

Comment:
The documentation here makes it look like the feature is already available - but it's actually not implemented yet.

Reply:
We broke the PR into smaller ones. That must be why it got slipped in. I will remove this change and submit a PR. I will add it back in the very last PR where I'm wiring everything up.

Reply:
thought the docs would be okay in the split up PR because the full implementation will be completed before the v0.43 release